packages/dcos-launch/extra/launch/aws.py

```diff
@@ -105,6 +98,12 @@ def delete_temp_resources(self, temp_resources):
             self.boto_wrapper.delete_vpc(temp_resources['vpc'])
 
     def key_helper(self, config):
+        """ If key_helper is true, then create an EC2 keypair with the same name
+        as the cloudformation stack, update the config with the resulting private key,
+        and amend the cloudformation template parameters to have KeyName set as this key
+        """
+        if config['key_helper'] != 'true':
+            return {}
```

Comment:
Same question as `zen_helper` - should throw an error when this condition is met? Is it safe to continue execution?